### PR TITLE
Add streaming load tests & workflows related to load tests

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
This adds the client, server, bidirectional streaming load tests and summary.csv PR creation, automated merging, automated load test invocation workflows

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/394

### CODE REVIEW

File: .github/workflows/create-summary-pr.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Create Automated Summary PR
```
Comment (BuddhiWathsala) [inline .github/workflows/create-summary-pr.yml:1]: Shall we elaborate the topic?. Something like Create Automated Summary PR for the Load Test.

File: load-tests/route_guide_bidirectional_streaming/deployment/deployment-patch.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+apiVersion: "apps/v1"
+kind: Deployment
+metadata:
+  name: no-name
```
Comment (BuddhiWathsala) [inline load-tests/route_guide_bidirectional_streaming/deployment/deployment-patch.yaml:4]: Don't we need a name here?.

File: load-tests/route_guide_bidirectional_streaming/results/summary.csv [hunk 3]
```diff
@@ -0,0 +1 @@
+Label,# Samples,Average,Median,90% Line,95% Line,99% Line,Min,Max,Error %,Throughput,Received KB/sec,Std. Dev.,Date,Payload,Users
```
Comment (BuddhiWathsala) [inline load-tests/route_guide_bidirectional_streaming/results/summary.csv:1]: Seems this is a generated CSV. Can't we add a trailing newline when this is being generated?

Comment (dilanSachi) [reply load-tests/route_guide_bidirectional_streaming/results/summary.csv]: Added in https://github.com/ballerina-platform/module-ballerina-grpc/pull/490/commits/4735ce77a9784ecdac578c36813759626a0f4a37

File: .github/workflows/create-summary-pr.yml (hunk 1 above)
Comment (dilanSachi) [reply .github/workflows/create-summary-pr.yml]: Added in https://github.com/ballerina-platform/module-ballerina-grpc/pull/490/commits/f7e37371d95a24cdcb8e9d9d0c3f8e50704d90f7

File: load-tests/route_guide_bidirectional_streaming/deployment/deployment-patch.yaml (hunk 2 above)
Comment (BuddhiWathsala) [reply load-tests/route_guide_bidirectional_streaming/deployment/deployment-patch.yaml:4]: Can you check with them, I think we don’t need this.

Comment (dilanSachi) [reply load-tests/route_guide_bidirectional_streaming/deployment/deployment-patch.yaml:4]: Apparently, the deployment-patch is used to indicate the `imagePullPolicy: Always` config. As they said, its now fixed in `c2c`. Will remove this.

Comment (dilanSachi) [reply load-tests/route_guide_bidirectional_streaming/deployment/deployment-patch.yaml]: Removed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/490/commits/3a9a72758d48b27e0d247135dc32dcd471aebdf7

**Review by BuddhiWathsala — APPROVED** [review]

**Pull request merged**